README.md

```diff
@@ -97,7 +97,9 @@ up a PR and add it to the list.
 ### Installing CRI-O
 To install CRI-O, you can use your distributions package manager:
 
-Fedora, CentOS, RHEL, and related distributions:
+Fedora 30 and later:
+```sudo dnf module install cri-o```
```

Comment:
Did yum go away onF30?  @cgwalters  tries to keep us in line with yum....

Reply:
Fedora should be using dnf.

Reply:
yum does in fact work though!

Reply:
yeah it is a link to dnf. dnf should be preferred in newer versions of Fedora.
```
$ ls -l /usr/bin/yum
lrwxrwxrwx. 1 root root 5 Jul 23 00:07 /usr/bin/yum -> dnf-3
```